### PR TITLE
Make push tokens easier to configure and clearer in purpose.

### PR DESCRIPTION
**Pull request by abusuioc**
Allows for the configuration of app side push tokens in the resource bundle.

### CODE REVIEW

**Comment by abusuioc** [comment]
Reminder about this PR.

File: README.md [hunk 1]
```diff
@@ -9,18 +9,18 @@
...
+Java sample code that encapsulates the APIs of the HUAWEI Push Kit server. It provides many sample programs for your reference or usage.
 
 The following table describes packages of Java sample code.
 
 | Package | Description |
 | ---- | ---- |
-| examples | Sample code packages. Each package can run independently. |
-| messaging | Package where APIs of the HUAWEI Push Kit server are encapsulated. |
+| examples | Sample code classes. Each class can run independently. |
+| message | Model classes mirroring the JSON data exchanged with the HUAWEI Push Kit server APIs. |
```
Comment (ferPrieto) [inline README.md:23]: That's messaging, because it's how the directory's called. I agree about the description though

File: README.md [hunk 2]
```diff
@@ -35,14 +35,18 @@ The following table lists parameters to be set.
 
 | Parameter | Description |
 | ---- | ---- |
-| appid | App ID, which is obtained from app information. |
-| appsecret | Secret access key of an app, which is obtained from app information. |
+| appid | The App ID of the app receiving the push messages. The App ID can be obtained from the App information section in AppGallery. |
```
Comment (ferPrieto) [inline README.md:38]: 'of the app' is a bit redundant, because it's assumed the AppID is about the app itself

File: README.md [hunk 3]
```diff
@@ -35,14 +35,18 @@ The following table lists parameters to be set.
...
+| appsecret | Secret access key of the app receiving the push messages. The app secret can be obtained from the App information section in AppGallery. |
```
Comment (ferPrieto) [inline README.md:39]: Same here with the 'of the app'

File: src/main/java/com/huawei/push/examples/SendApnsMessage.java [hunk 4]
```diff
@@ -61,8 +63,10 @@ public void sendApnsMessage() throws HuaweiMesssagingException {
                 .setHmsOptions(apnsHmsOptions)
                 .build();
 
+        String apnsPushToken = ResourceBundle.getBundle("url").getString("apns_token");
+
         Message message = Message.builder().setApns(apns)
-                .addToken("9FDA406A04BDE017A2F53EB9831846FBF5308567DE9A4E986D96512136F72C3D")
+                .addToken(apnsPushToken)
```
Comment (ferPrieto) [inline src/main/java/com/huawei/push/examples/SendApnsMessage.java:69]: 👍

**Review by ferPrieto — CHANGES_REQUESTED** [review]
Looking good overall, there's just a couple of minor things in the text that I would change.

File: README.md (hunk 1 above)
Comment (abusuioc) [reply README.md:23]: Actually the model classes are located under "message".

**Review by ferPrieto — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ferPrieto) [reply README.md:23]: Yeah I realised after double checking my other comments

Comment (ferPrieto) [reply README.md:23]: But we would need to keep the messaging description too then

**Pull request closed without merging**